lib/kitchen/directions/bake_numbered_exercise/v1.rb

```diff
@@ -2,15 +2,20 @@
 
 module Kitchen::Directions::BakeNumberedExercise
   class V1
-    def bake(exercise:, number:, suppress_solution: false)
+    def bake(exercise:, number:, suppress_solution: false, suppress_even_solution: false)
       problem = exercise.problem
       solution = exercise.solution
-
       problem_number = "<span class='os-number'>#{number}</span>"
 
       if solution.present?
         if suppress_solution
           solution.trash
+        elsif suppress_even_solution && number.odd?
+          problem_number = "<a class='os-number' href='##{exercise.id}-solution'>#{number}</a>"
+          bake_solution(exercise: exercise, number: number)
+        elsif suppress_even_solution && number.even?
+          problem_number = "<span class='os-number'>#{number}</span>"
+          bake_even_solution(exercise: exercise)
```

Comment:
In Sociology only odd solutions are needed in Answer Key, so I suppressed even ones based on the number of the exercises - it was needed to be done in this file because in that case the link pointing to solution can be only present in the in the odd exercises numbers.

Reply:
I made a suggestion in https://github.com/openstax/kitchen/pull/243

Reply:
@jpslav Thank you for this idea! :slightly_smiling_face: There were some small fixes need in the PR you have created to make sociology recipe be baked with your changes, but it really simplified things.